content/kubernetes/preview/_index.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Is this file (and all other AA-related) files) purposely included in this PR?

Reply:
No that is not on purpose. I'll fix that.